### PR TITLE
Port staking changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description

Port staking changes to Chinese docs site.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

https://github.com/PureStake/moonbeam-docs/pull/215
https://github.com/PureStake/moonbeam-docs/pull/217
https://github.com/PureStake/moonbeam-docs/pull/219

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

None

### CODE REVIEW

File: node-operators/networks/collator.md [hunk 1]
```diff
@@ -33,19 +33,23 @@ Moonbeam使用[Nimbus平行链共识框架](/learn/features/consensus/)，通过
...
-和波卡（Polkadot）验证人相似，收集人也需要创建账户。Moonbeam使用的是拥有私钥的H160账户或以太坊式账户。另外，需要拥有最低Token质押量才有资格成为候选人。只有一定数量的根据提名质押量排名靠前的收集人才会进入收集人有效集。
+和波卡（Polkadot）验证人相似，收集人也需要创建账户。Moonbeam使用的是拥有私钥的H160账户或以太坊式账户。另外，需要拥有高于最低自质押量(Self-bonded)才有资格成为候选人。只有一定数量的根据质押量和委托量排名靠前的收集人才会进入收集人有效集。
 
 === "Moonriver"
     |    变量     |                           值                           |
     |:---------------:|:---------------------------------------------------------:|
-    |   绑定数量   | {{ networks.moonriver.staking.min_can_stk }}枚MOVR  |
-    | 有效集上限 | {{ networks.moonriver.staking.max_candidates }}名收集人 |
+    |   绑定数量   | {{ networks.moonriver.staking.min_can_stk }} MOVR  |
+    | 最小自质押量 |     {{ networks.moonriver.staking.min_can_stk }} MOVR     |
```
Comment (eshaben) [inline node-operators/networks/collator.md:42]: looks like these use the same variables, only one is needed for Minimum self-bond amount

File: node-operators/networks/collator.md [hunk 2]
```diff
@@ -33,19 +33,23 @@ Moonbeam使用[Nimbus平行链共识框架](/learn/features/consensus/)，通过
...
+    | 最小总质押量 |     {{ networks.moonriver.staking.min_col_stk }} MOVR     |
+    | 有效集上限 | {{ networks.moonriver.staking.max_candidates }} |
 
 === "Moonbase Alpha"
     |    变量     |                          值                           |
     |:---------------:|:--------------------------------------------------------:|
-    |   绑定数量   |  {{ networks.moonbase.staking.min_can_stk }}枚DEV  |
-    | 有效集上限 | {{ networks.moonbase.staking.max_candidates }}名收集人 |
+    |   绑定数量   |  {{ networks.moonbase.staking.min_can_stk }} DEV  |
+    | 最小自质押量 |     {{ networks.moonbase.staking.min_can_stk }} DEV    |
```
Comment (eshaben) [inline node-operators/networks/collator.md:50]: same comment as above

**Review by eshaben — CHANGES_REQUESTED** [review]
Just a couple minor fixes are needed! Other than that it looks good ✨

**Comment by hyd628** [comment]
Good catch, thanks! The redundant entry is removed.

**Review by eshaben — APPROVED** [review]
sweeet LGTM! 🚀

**Pull request merged**